apps/institution-service/src/controllers/institutionController.ts

```diff
@@ -191,9 +201,34 @@ const getPaginationOptions = (req: Request): PaginationOptions => {
   const limit =
     parseInt(req.query.pageSize as string, 10) || DEFAULT_PAGINATION_PAGE_SIZE;
   const offset = (page - 1) * limit;
+
   return { page, limit, offset };
 };
 
+const parseSort = (sortBy: string[]): SortSequelize[] => {
+  return sortBy.map((param) => {
+    const [column, direction = SortDirection.ASC] = param.split(":");
+    return { column, direction: direction as SortDirection };
+  });
+};
+
+export const getSortOption = (
```

Comment:
This looks like a premature abstraction to me. We don't have a second use case for this function, but it's built like it wants to be reused.

Abstracting is fine if it's only meant to be used by this one function, but then it should be built like that. DefaultSortString isn't necessary, because there's only one use case right now.